typescript/cli/ci-test.sh

```diff
@@ -1,18 +1,47 @@
 #!/usr/bin/env bash
 
-# NOTE: This script is intended to be run from the root of the repo
-
-# the first arg to this script is a flag to enable the hook config as part of core deployment
-# motivation is to test both the bare bone deployment (included in the docs) and the deployment with the routing over igp hook (which is closer to production deployment)
-HOOK_FLAG=$1
-if [ -z "$HOOK_FLAG" ]; then
-  echo "Usage: fork.sh <hook>"
-  exit 1
+# set script location as repo root
+cd "$(dirname "$0")/../.."
```

Comment:
Just to confirm, this still works when running this from root?

Reply:
yup, this change just means you can run from anywhere and it'll auto-orient itself to the repo root for the script